cmd/cdi-importer/BUILD.bazel

```diff
@@ -18,6 +18,7 @@ rpm_image(
             "@nbdkit-xz-filter//file",
             "@nbdkit-gzip-filter//file",
             "@qemu-img-aarch64//file",
+            "@qemu-block-curl-aarch64//file",
```

Comment:
Why are we adding block curl, but just for arm?

Reply:
Leftover, removed.